healthyregions/SDOHPlace
# feat: first-pass at Guides routes/page

## Problem
We need a new page / set of pages to present Guides to the user

Our first guide will be Public Transit Equity

Fixes #318 

## Approach
* feat: copy existing PostLayout to use as a base for Guides section
* fix: cut some corners on author/last_updated/body to get the page to render
* feat: added carousel styling/behavior to homepage

## How to Test
1. Navigate to https://deploy-preview-323--cheerful-treacle-913a24.netlify.app
2. Scroll down to the new carousel displaying SDOH factors (Transit, Greenspaces, Education, Workspaces, etc)
    * You should see a Learn More button here for "Transit"
3. Click on the "Transit" factor
    * You should be taken to the new guide on Public Transit Equity
